rubocop-rails.yml

```diff
@@ -12,3 +12,94 @@ Rails/EnumUniqueness:
 
 Rails/ReversibleMigrationMethodDefinition:
   Enabled: true
+
+Rails/ActiveSupportAliases:
+  Enabled: true
+
+Rails/AddColumnIndex:
+  Enabled: true
+  AutoCorrect: false
```

Comment:
Why not enable the autocorrection?

Reply:
Ok, I thought it would be something along those lines :slightly_smiling_face: 
I guess we’ll need to fix those indexes anyway, right?
So we could allow rubocop to change the migrations and use [`add_index(:table, :name, if_not_exists: true)` ](https://api.rubyonrails.org/v8.0.3/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_index-label-Creating+a+index+which+already+exists), wdyt?